bin/alias.bat

```diff
@@ -1,27 +1,31 @@
 @echo off
 
 set ALIASES=%CMDER_ROOT%\config\aliases
+setlocal
+:: handle quotes within command definition, e.g. quoted long file names
+set _x="%*"
+set _x=%_x:"=%
 
-if ["%*"] == [""] echo Use /? for help & echo. & goto :p_show
+:: check command usage
+if ["%_x%"] == [""] echo Use /? for help & echo. & goto :p_show
```

Comment:
The reason %_x% is used here instead of %* is because of the surrounding quotes. If you use `if ["%*"] == [""]` and try to run the following alias `alias runapi=runas /netonly /user:domain\user "c:\Program Files (x86)\IIS Express\iisexpress.exe /site :WebAppName /trace:i"` you will receive the error "Files was unexpected at this time", because the quotes surrounding %* get mismatched with the quotes around "c:\Program Files...".